projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -61,6 +61,9 @@ public static function load_rest_api_endpoints() {
 	 * @param string ...$plugins List of plugins to update.
 	 */
 	public static function run_scheduled_update( ...$plugins ) {
+		// Save the last run schedule ID.
+		set_site_transient( 'jetpack_scheduled_update_last_run', self::generate_schedule_id( $plugins ) );
```

Comment:
What if we passed the schedule id to the plugin updates endpoint to give `Jetpack_Autoupdate` an ID to talk to?

Reply:
Yes, we would. I didn't want to touch too many things. Passing a new field means modifying the endpoint, passing it to `run_plugin_autoupdates`, to the job (in two steps), and others. A simple transient solved all this. I think this is a good addition to do later.